firestore.rules

```diff
@@ -89,6 +89,32 @@ service cloud.firestore {
       return userIsMemberOfParent;
     }
 
+    /**
+     * Return true if the current user is a member of the parent of the
+     * document's objective *before* performing the action.
+     */
```

Comment:
Har du sett noe rundt syntaks for kommentarer i [CEL](https://firebase.google.com/docs/rules/rules-language)? Er det bare `//` som [egentlig er akseptert](https://github.com/google/cel-spec/blob/master/doc/langdef.md#syntax)? (Min syntax-highlighter er ikke så glad, men er heller ikke så veldig farlig.)

Reply:
Ah, hm. Ser ut som de andre kommentarene blir ignorert også, men da bytter jeg dem om.